### PR TITLE
tegra-tests: add nvidia-docker-tests

### PR DESCRIPTION
**Pull request by atharvanan1**
- This will test container compatibility on the device image
- Tested on `jetson-xavier-nx-devkit-emmc`

### CODE REVIEW

File: recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+DESCRIPTION = "Scripts for testing docker containers with PyTorch, TensorFlow, TensorRT and CUDA library"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=da33f1c12f6d31e71cd114d0c336d4be"
+
+SRC_REPO ?= "github.com/atharvanan1/jetson-containers"
```
Comment (dwalkes) [inline recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb:5]: @madisongh should this live in an OE4T repo before https://github.com/dusty-nv/jetson-containers/pull/189 is approved?

Comment (atharvanan1) [reply recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb:5]: We can shift it there. If you can stage it there, I'll change the SRC_REPO accordingly.

**Comment by dwalkes** [comment]
> Tested on jetson-xavier-nx-devkit-emmc

I think you need some tricks to get a `/var` partition greater than 16GB on this platform, correct?

**Comment by atharvanan1** [comment]
Ah yes. I needed to space for pulling docker images.

**Comment by dwalkes** [comment]
Trying to figure out where we should document this.  There are instructions in https://docs.google.com/spreadsheets/d/1EWG5QZvtWdk74gz48D4xCbegK-bsqMFJ8XOsyGdUUeI/edit#gid=593862884 but those suggest using the devkit build.

**Comment by atharvanan1** [comment]
> those suggest using the devkit build.

Do you mean the sdcard machine?

**Comment by dwalkes** [comment]
> Do you mean the sdcard machine?
Yes, `devkit` and not `devkit-emmc`

File: recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb (hunk 1 above)
Comment (madisongh) [reply recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb:5]: The changes look pretty small... any chance you could just generate a patch to apply to the upstream sources until the PR is resolved?

Comment (atharvanan1) [reply recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb:5]: Noted! Thanks for you input.

**Comment by atharvanan1** [comment]
> Yes, devkit and not devkit-emmc

For this, I tried to use the extra space available on the sdcard. Ex. Using 32 GB SDcard, provided me with 16GB of extra space.

File: recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb (hunk 1 above)
Comment (atharvanan1) [reply recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb]: Done.

**Comment by atharvanan1** [comment]
Should be updated with a small patch now.

**Comment by atharvanan1** [comment]
Thanks guys for your help!

**Pull request merged**